src/pages/edit/index.vue

```diff
@@ -100,6 +119,16 @@ export default Vue.extend({
     },
     dateTitle() {
       return dayjs(vxm.app.currentDate).format('M/D')
+    },
+    lessonsGroupByPeriod() {
+      const groupBy = (targets: LessonWithId[], key: keyof LessonWithId) =>
+        targets.reduce((acc: LessonsGroupedBy, currentLesson: LessonWithId) => {
+          const valueToGroup = currentLesson[key].toString()
+          acc[valueToGroup] = acc[valueToGroup] || []
+          acc[valueToGroup].push(currentLesson)
+          return acc
+        }, {})
+      return groupBy(this.classData.lessonsOnCurrentDate, 'startTime')
```

Comment:
```suggestion
      return groupBy(vxm.classData.lessonsOnCurrentDate, 'startTime')
```

こうするとtypeErrorは消えますね（正しい対応かどうかの自信がないです）

Reply:
おお、そうなのですね！では、ひとまずこれでいきますかね。